prog/postgres/postgres_server_nexus.rb

```diff
@@ -653,6 +653,13 @@ def before_run
 
   label def restart
     decr_restart
+
+    if postgres_server.primary?
```

Comment:
Why do we use different deadlines for primary and standby?

Reply:
I took the same pattern we use for setting deadlines in `bootstrap_rhizome` here